go/common/sgx/pcs/quote.go

```diff
@@ -31,8 +31,21 @@ const (
 
 	// ppidDataLen is the PPID certification data length in bytes.
 	ppidDataLen = 404
+
+	// DefaultMinTCBEvaluationDataNumber is the default minimum TCB evaluation data number.
+	DefaultMinTCBEvaluationDataNumber = 12 // As of 2022-08-01.
 )
 
+// QuotePolicy is the quote validity policy.
+type QuotePolicy struct {
+	// TCBValidityPeriod is the validity (in days) of the TCB collateral.
+	TCBValidityPeriod uint16 `json:"tcb_validity_period"`
```

Comment:
:information_source: previously we treated it as valid until the "next update" time